packages/gateway/src/types.ts

```diff
@@ -68,3 +68,32 @@ export interface IProofService<T extends ProvableBlock> {
    */
   getProofs(block: T, address: Address, slots: bigint[]): Promise<Hash>
 }
+
+/**
+ * Interface for a block cache.
+ */
+export interface IBlockCache {
```

Comment:
this could be moved to `services` given that it is only used there. This `types` file is more for variables that are used throughout the whole codebase